content/en/users/getting-started/_index.md

```diff
@@ -3,9 +3,124 @@ title: "Getting Started"
 linkTitle: "Getting Started"
 type: docs
 weight: 5
-draft: true
 description: >
-  Introduction to using EGI services
+  Introduction to EGI FedCloud
 ---
 
-TBD
+## Overview
+
+EGI is a federation of computing and storage resource providers
+united by a mission to support research and innovation.
+
+The resources in the EGI Federated Cloud (FedCloud) are offered by
```

Comment:
I agree. But this PR (which I cooked for a long time) is to reorganize the info that is already in the docs, so that the same thing is not described over and over again. Please propose improvements. The only thing I fear is that doing both the reorg and add new content in the same PR will turn out to be a mess to work with.

Reply:
OK, thought there was also new additions/rephrasing and so on.
I'm fine with doing the name checking in a further iteration.

Reply:
But still the confusion/assumption that everything is Cloud is a bit problematic/misleading. May be just a few sentences/a short section may help.

Reply:
It is misleading, because I am only now grasping that, after 3 months. However, maybe a discussion about what "cloud" means would be welcome. For me, everything that is not on-prem is cloud.

Reply:
I would say that in our community we have a different view/history: initially all the distributed computing started with Grid computing (AKA HTC) using specific middleware and components (for distributed computing and storage), and some years ago there a shift to cloud started, leveraging CMF like OpenStack and OpenNebula to provide IaaS and cloud APIs/services.
Thus gird/HTC and cloud are usually still seen as two very different things. But it's also true that the limits are now a moving target and more blurry, having grid sites deployed on top or cloud (and grid sites allowing users to run containers).